hyp3_gamma/dem.py

```diff
@@ -99,11 +119,14 @@ def prepare_dem_geotiff(output_name: str, geometry: ogr.Geometry, pixel_size: fl
         with TemporaryDirectory() as temp_dir:
             temp_path = Path(temp_dir)
 
+            geometry = get_envelope_geometry(geometry)
```

Comment:
Why can't we just do the following?

```python
geometry = geometry.GetEnvelope()
```

Reply:
For geometry does not cross over anti meridian, we use  geometry = geometry.GetEnvelope(), but this does not work for gemoetry that croses over anti-meridian, This is why I developed the function for the purpose.

Reply:
That makes sense, thanks for the clarification. Can you add some unit tests for this function? We should probably have one test for a geometry that does not cross the anti-meridian, and another test for a geometry that crosses the anti-meridian.